vuepress/docs/next/tutorials/getting-started/kubernetes-install.md

```diff
@@ -2,45 +2,47 @@
-### For Entando install only:
+Minimum hardware requirements depend on how Entando is used. As a general rule, an increase in functionality necessitates additional capabilities and resources.
+### Entando Install Only
 
-- 6 vCPU or virtual central processing unit  (6000 millicores)
+- 6 vCPU (6000 millicores)
 - 6 GB of RAM
-- 5 GB of allocated storage distributed in many PVCs
+- 5 GB of allocated storage distributed over many PVC
 
-Configure a single worker node with the requirements above or designate 3 worker nodes with the minimum of 2 PVC and 4 GB of RAM each.
+Configure a single worker node with the requirements above or designate 3 worker nodes, each with a minimum of 2 PVC and 4 GB of RAM.
 
-### For Entando + Microservices:
+### Entando + Microservices
 
-- 500 millicores per microservice or 
-- 1000 millicores per microservice with a database
+- 500 millicores per microservice, or 
+- 1000 millicores per microservice containing a database
 
-The tested instance was generated with a JHipster Entando blueprint.
+The microservices tested with the CPU specs above were generated using the JHipster Entando Blueprint.
 
-### For Entando with CMS: 
+### Entando + CMS
 
-To install Entando with CMS, the hardware requirements depend on the complexity and number of pages and contents. The minimum requirements are:
+Complexity, content and number of pages determine the hardware requirements of an Entando install that includes the CMS. The minimum requirements are:
 
-- 9 vCPU for a single CMS instance. To scale up, you need 4 vCPU per replica.
-- 8 GB of RAM for a single CMS instance. To scale up, you need 4 GB of RAM per replica.
-- A minimum of 50 GB of storage for all the static resources generated by the use of Entando CMS.
+- 9 vCPU for a single CMS instance. To scale up, allocate 4 vCPU per replica.
+- 8 GB of RAM for a single CMS instance. To scale up, allocate 4 GB of RAM per replica.
+- A minimum of 50 GB of storage for the static resources generated by the use of the Entando CMS.
 
 ## Backup Strategies
 
-The recommended method to secure your content is to use database services that automate backups and restores on a Kubernetes cluster like Azure, Red Hat OpenShift, Google K8s Service or Amazon Elastic. If this is not an option, there are tools like [Velero](https://velero.io/) and [Cloudcase](https://cloudcasa.io/) to manage them. 
+It is recommended to secure content via database services that automate backups and restore on a Kubernetes cluster such as Azure, Red Hat OpenShift, Google K8s Service or Amazon Elastic. If this is not an option, content can be managed with tools like [Velero](https://velero.io/) or [Cloudcase](https://cloudcasa.io/). 
 
-Note: Some DevOps teams warn against running databases inside pods in production environments.
+> Note: The decision to run databases inside production environment pods is controversial.
```

Comment:
does this change the meaning at all?  are pods inside the production environment the same as production environment pods?

Reply:
I don't think it's quite the same. It's not a controversial topic per se, some DevOps teams prefer one versus the other. Controversy somehow applies moral ambiguity to me and that isn't the case here.  Maybe "Note: Some DevOps teams recommend against running production databases inside pods since it makes database management more challenging."